docker-for-windows/release-notes.md

```diff
@@ -13,6 +13,22 @@ This page contains information about the new features, improvements, known issue
 For information about Edge releases, see the [Edge release notes](edge-release-notes). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.2.0.4
+2020-03-
+
+### Bug fixes and minor changes
+
+- Docker Desktop now displays hidden files in shared volumes. Fixes [docker/for-win#5808](https://github.com/docker/for-win/issues/5808).
+- Docker Desktop now generates the inotify `MODIFY` events on shared filesystems for Windows file sharing. Fixes [docker/for-win#5530](https://github.com/docker/for-win/issues/5530).
```

Comment:
nit: maybe `file systems` instead of `filesystems`? Or have we referred to `filesystems` in the past?

Reply:
Updated.